avm/res/document-db/database-account/sql-database/container/main.bicep

```diff
@@ -62,7 +62,7 @@ var containerResourceParams = union(
     partitionKey: {
       paths: partitionKeyPaths
       kind: kind
-      version: kind == 'MultiHash' ? 2 : 1
+      version: kind == 'Range' ? 1 : 2
```

Comment:
Hey @Agazoth thanks for your contribution. 
Please run the set-AVMModule utility to reflect the changes in the compiled main.json.
Also please attach a pipeline badge to prove the successful run of the module pipeline after this change.

CC @bryansan-msft for further review

Reply:
Done